pkgs/tools/networking/sslsplit/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, openssl, libevent, libpcap, libnet, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "sslsplit-${version}";
+  version = "0.5.4";
+
+  src = fetchurl {
+    url = "https://github.com/droe/sslsplit/archive/${version}.tar.gz";
+    sha256 = "1zi9xyh2wfi5jszq69f3fjfjmkvmrqxryq34qmnaza60k1jjaf1k";
+  };
+
+  buildInputs = [ openssl libevent libpcap libnet zlib ];
+
+  makeFlags = [
+    "DESTDIR=$(out)"
+    "PREFIX="
+    "OPENSSL_BASE=${openssl.dev}"
+    "LIBEVENT_BASE=${libevent.dev}"
+    "LIBPCAP_BASE=${libpcap}"
+    "LIBNET_BASE=${libnet}"
+  ];
+
+  meta = with stdenv.lib; {
```

Comment:
Would you be interested in owning this package, maybe?

Reply:
It would be lovely. Does this mean adding a maintainer field?

Reply:
Great! Please be sure to add yourself to the maintainer list in a separate commit.